upload/admin/view/template/module/smaily_for_opencart.tpl

```diff
@@ -321,6 +317,74 @@
+                    </select>
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="col-sm-2 control-label" for="rss-sort-by"><?php echo $rss_sort_by_title; ?></label>
+                <div class="col-sm-10">
+                  <select name="smaily_for_opencart_rss_sort_by" id="rss-sort-by" class="form-control smaily-rss-options">
+                   <?php
+                    // Add options for select.
+                    foreach ($sort_options as $sort_code => $sort_name) {
+                      $selected = $rss_sort_by === $sort_code ? 'selected' : ''; ?>
+                      <option value="<?php echo $sort_code; ?>"<?php echo $selected; ?>><?php echo $sort_name; ?></option>
+                    <?php } ?>
+                    </select>
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="col-sm-2 control-label" for="rss-sort-order"><?php echo $rss_sort_order_title; ?></label>
+                <div class="col-sm-10">
+                  <select name="smaily_for_opencart_rss_sort_order" id="rss-sort-order" class="form-control smaily-rss-options">
+                    <?php if ($rss_sort_order == 'ASC') { ?>
+                    <option value="ASC" selected="selected"><?php echo $text_ascending; ?></option>
+                    <option value="DESC"><?php echo $text_descending; ?></option>
+                    <?php } else { ?>
+                    <option value="ASC"><?php echo $text_ascending; ?></option>
+                    <option value="DESC" selected="selected"><?php echo $text_descending; ?></option>
+                    <?php } ?>
+                  </select>
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="col-sm-2 control-label" for="smaily_for_opencart_rss_limit"><?php echo $rss_limit_title; ?></label>
+                <div class="col-sm-10">
+                  <div class="input-group">
+                    <input type="number"
+                          name="smaily_for_opencart_rss_limit"
+                          min="1"
+                          max="250"
+                          id="rss-limit"
+                          value="<?php echo $rss_limit; ?>"
+                          class="form-control smaily-rss-options" />
+                    <span class="input-group-addon"><?php echo $rss_limit_products; ?></span>
+                  </div>
+                  <?php if ($error_limit) { ?>
+                      <div class="text-danger"><?php echo $error_limit; ?></div>
+                  <?php } ?>
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="col-sm-2 control-label"><?php echo $rss_feed_title; ?></label>
+                <div class="col-sm-10">
+                  <p><strong id="smaily-rss-feed-url" name="rss_feed_url"><?php echo $smaily_rss_url; ?></strong></p>
```

Comment:
HTML's `<strong>` tag does not have `name` attribute. What is its purpose?

Reply:
No purpose, removed.